src/pages/api/track.ts

```
@@ -0,0 +1,63 @@
+// pages/api/track.ts
+
+import { NextApiRequest, NextApiResponse } from 'next';
+import prisma from '../../utils/prisma';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method !== 'POST') {
+    return res.status(405).json({ message: 'Method Not Allowed' });
+  }
+
+  const sanitize = (input: any) =>
+    typeof input === 'string' ? input.replace(/\u0000/g, '').trim() : input;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)